apcd-cms/src/apps/view_users/views.py

```diff
@@ -36,93 +48,72 @@ def _edit_user(form):
-                    'user_name': usr[3],
-                    'org_name': usr[4],
-                    'created_at': usr[5],
-                    'updated_at': usr[6],
-                    'notes': usr[7],
-                    'status': 'Active' if usr[8] else 'Inactive',
-                    'user_number': usr[9],
-                    'role_name': usr[10],
-                    'org_name_no_parens': usr[4].replace("(", "").replace(")", ""),  # just for filtering purposes
-                    'active': usr[8],
-                }
-
-        context['header'] = ['User ID', 'Name', 'Organization', 'Role', 'Status', 'User Number', 'See More']
+        context['header'] = ['User ID', 'Name', 'Organization', 'Role', 'Active', 'User Number', 'See More']
         context['status_options'] = ['All', 'Active', 'Inactive']
-        context['filter_options'] = ['All']
-        context['role_options'] = ['SUBMITTER_USER', 'SUBMITTER_ADMIN','APCD_ADMIN']
-
-        context['status'] = ['Active', 'Inactive']
+        context['role_options'] = ['SUBMITTER_USER', 'SUBMITTER_ADMIN', 'APCD_ADMIN']
+        context['status'] = ['True', 'False']
+        context['filter_options'] = ['All'] + get_api_users_org_list()
 
         # this kind of sucks, we should make this not hard coded, just getting it to work for now
         context['roles'] = [
             {'role_id': 1, 'role_name': 'APCD_ADMIN'},
             {'role_id': 2, 'role_name': 'SUBMITTER_ADMIN'},
             {'role_id': 3, 'role_name': 'SUBMITTER_USER'}
         ]
-        
+
         try:
             page_num = int(self.request.GET.get('page'))
         except:
-            page_num = 1        
-        table_entries = []
-        for user in user_content:
-            table_entries.append(_set_user(user,))
-            org_name = user[4]
-            if org_name not in context['filter_options']:  # prevent duplicates
-                context['filter_options'].append(user[4])
+            page_num = 1
 
         queryStr = ''
         status_filter = self.request.GET.get('status')
         org_filter = self.request.GET.get('org')
         role_filter = self.request.GET.get('role_name')
 
         context['selected_status'] = None
-        if status_filter is not None and status_filter !='All':
+        if status_filter is not None and status_filter != 'All':
             context['selected_status'] = status_filter
             queryStr += f'&status={status_filter}'
-            table_entries = table_filter(status_filter, table_entries, 'status', False)
+            # table_entries = table_filter(status_filter, table_entries, 'status', False)
 
-        if role_filter is not None and role_filter !='All':
+        if role_filter is not None and role_filter != 'All':
             context['selected_role'] = role_filter
-            table_entries = table_filter(role_filter, table_entries, 'role_name', False)        
+            # table_entries = table_filter(role_filter, table_entries, 'role_name', False)
```

Comment:
Do we want to keep the commented lines of code in this file diff?

Reply:
I will remove those

Reply:
@rstijerina , comments removed